addon/controllers/scopes.js

```diff
@@ -0,0 +1,17 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+
+export default class ScopesController extends Controller {
+  @service router;
+
+  get activeScope() {
+    if (!this.router.currentRouteName.includes("ember-emeis.scopes.edit")) {
```

Comment:
this feels not right. What do you think @czosel ?

Reply:
@velrest and I just took another look at this. We didn't find a good alternative, but we did find some documentation for `attributes`: https://emberjs.github.io/rfcs/0095-router-service.html#routeinfowithattributes
I'd stay let's stick with it :slightly_smiling_face: